spec/index.bs

```diff
@@ -1566,6 +1567,7 @@ dictionary GPUBindGroupLayoutBinding {
     GPUTextureComponentType textureComponentType = "float";
     boolean multisampled = false;
     boolean hasDynamicOffset = false;
+    GPUTextureFormat storageTextureFormat = {};
```

Comment:
The default value `{}` is only for dictionaries. You don't need to provide a default value; then the default value will be "not set" (`undefined`).

Reply:
Done